File: Client/Android/app/src/main/java/com/example/examinapp/dataaccess/dtos/login/LoginRequest.java
package com.example.examinapp.dataaccess.dtos.login;

import com.example.examinapp.dataaccess.dtos.base.BaseRequest;

import java.util.Date;

public class LoginRequest extends BaseRequest {
    private String password;

    public LoginRequest(Date requestedDateTime, String username, String sessionId, String password) {
        super(requestedDateTime, username, sessionId);
        this.password = password;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: Server/CTSE-Exam-In-App/DTOs/Login/LoginRequest.cs
using CTSE_Exam_In_App.DTOs.Base;
using CTSE_Exam_In_App.Models;

namespace CTSE_Exam_In_App.DTOs.Login
{
    public class LoginRequest : BaseRequest
    {
        public string Password { get; set; }
    }
}
